lib/data/repository/movies_repository_impl.dart

```diff
@@ -35,9 +40,14 @@ class MoviesRepository with DatabaseResponse implements IMoviesRepository {
         var apiResponse = await _service.apiCall(endpoint: endpoint);
         if (apiResponse.statusCode == HttpStatus.ok) {
           List<dynamic> movies = json.decode(apiResponse.body)['results'];
+
           if (movies.isEmpty) {
             return MovieEvent(status: Status.empty);
           } else {
+            movies.forEach((movie) {
+              (movie as Map<String, dynamic>)['lowerCaseTitle'] =
```

Comment:
'results' and 'lowerCaseTitle' can be added to a constant value

Reply:
Done